### PR TITLE
Fix/tracking section and bookmark widget

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
`tracking-section` for article page & `bookmark-widget` render issue while cause article page crashed

# Notice

# Dependency

### CODE REVIEW

**Review by Aylie-Chou — APPROVED** [review]
LGTM

File: packages/react-components/src/bookmark-widget/index.js [hunk 1]
```diff
@@ -98,7 +115,7 @@ const BookmarkWidget = ({ articleMeta, renderIcon, toAutoCheck = true }) => {
         articleSlug
       )
     }
-  }, [articleMeta, isAuthed, toAutoCheck, jwt, userID, dispatch])
+  }, [])
```
Comment (Aylie-Chou) [inline packages/react-components/src/bookmark-widget/index.js:118]: Would we miss to check bookmark status when clicking related post if we remove the dependency of `articleMeta` here ?

**Review by Aylie-Chou — COMMENTED** [review]
Just have a question about the dependency of useEffect
Please have a look at comment ~~

File: packages/react-components/src/bookmark-widget/index.js (hunk 1 above)
Comment (liruchen32) [reply packages/react-components/src/bookmark-widget/index.js:118]: 不會，因為進到相關文章後這個 component 會重新 mount 一次

Comment (Aylie-Chou) [reply packages/react-components/src/bookmark-widget/index.js:118]: Okay~~

**Review by Aylie-Chou — APPROVED** [review]
LGTM*2

**Pull request merged**